packages/cloud/src/schema/player.ts

```diff
@@ -0,0 +1,43 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+
+import { CombatUnitSpec } from '../../../parser/dist/index';
+import { TeamStatRecord } from './team';
+
+@Entity()
+export class PlayerStatRecord {
+  @PrimaryGeneratedColumn()
+  rowId: number;
+
```

Comment:
On a similar note -- is it worth adding IsUploader or IsUploaderTeam flags to these?

Reply:
I'll try to build a few queries - I'm expecting the ORM to handle joins easily in which case we would be able to check those conditions. Worst case we'll write SQL statements ourselves to do the joins.